src/pharmpy/modeling/summary_individuals.py

```diff
@@ -0,0 +1,108 @@
+from typing import Union, Callable
+
+import pandas as pd
+import numpy as np
+
+from pharmpy.tools.modelsearch.tool import ModelSearchResults
+from pharmpy.model import Model
+
+from pharmpy.modeling.ml import predict_outliers
+from pharmpy.modeling.ml import predict_influential_individuals
+
+def summary_individuals(results: ModelSearchResults) -> Union[pd.DataFrame,None]:
+    """Creates a summary dataframe keyed by model-individual pairs for an input
+    ModelSearchResults object.
+
+    Parameters
+    ----------
+    results : ModelSearchResults
+        Input results object
+
+    Return
+    ------
+    pd.DataFrame | None
+        The summary as a dataframe
+
+    Examples
+    --------
+    >>> from pharmpy.modeling import *
+    >>> model = load_example_model("pheno")
+    >>> fit(model)
+    >>> results = run_tool('modelsearch', model=model, mfl='ABSORPTION(ZO);PERIPHERALS([1, 2])', algorithm='reduced_stepwise')
+    >>> summary_individuals(results) # doctest: +SKIP
+
+    """
+
+    if results.start_model is None: return None
+
+    models = [results.start_model] if results.models is None else [results.start_model, *results.models]
+
+    df = pd.concat(map(
+        groupedByIDAddColumnsOneModel,
+        models,
+    ), keys=[model.name for model in models], names=['model'], axis=0, verify_integrity=True)
+
+    df['diOFV'] = df.apply(lambda row: row['iOFV'] - df.loc[(row['parent_model'], row.name[1])]['iOFV'], axis=1)
+
+    return df
+
+def constant_series(model: Model, name: str, value) -> pd.Series:
+    index = model.dataset['ID'].unique()
+    series = pd.Series(np.repeat(value, len(index)), index=index, name=name)
+    series.index.rename('ID', inplace=True)
+    return series
+
+def parent_model_name(model: Model) -> pd.Series:
+    return constant_series(model, 'parent_model', model.parent_model)
+
+def model_name(model: Model) -> pd.Series:
+    return constant_series(model, 'model', model.name)
+
+def outliers_fda_func(df: pd.DataFrame) -> float:
+    # NOTE this returns a float because we will potentially concat this
+    # with NaNs
+    return float((abs(df) > 5).sum())
```

Comment:
`5` or the entire condition could be made parameterizable

Reply:
We could leave as 5 for now